### PR TITLE
Unit Test Cleanup

### PR DESCRIPTION
**Pull request by david-waltermire**
# Committer Notes

Adjusted schema generation unit tests to ensure that each test has a unique XML and JSON URI and root element name.

Removed duplicate token test that were outside the correct location, moving the test vectors to the proper location. 

Removed stored schemas for tests, which are not tested and not well maintained.

### All Submissions:

- [ ] Have you followed the guidelines in our [Contributing](https://github.com/usnistgov/metaschema/blob/master/CONTRIBUTING.md) document?
- [ ] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/usnistgov/metaschema/pulls) for the same update/change?
- [ ] Have you squashed any non-relevant commits and commit messages? \[[instructions](https://git-scm.com/book/en/v2/Git-Tools-Rewriting-History)\]
- [ ] Do all automated CI/CD checks pass?

### Changes to Core Features:

- [ ] Have you added an explanation of what your changes do and why you'd like us to include them?
- [ ] Have you written new tests for your core changes, as applicable?
- [ ] Have you included examples of how to use your new feature(s)?
- [ ] Have you updated all website](https://pages.nist.gov/metaschema) and readme documentation affected by the changes you made? Changes to the website can be made in the website/content directory of your branch.

### CODE REVIEW

File: test-suite/schema-generation/group-as/group-as-array-bounded_metaschema.xml [hunk 1]
```diff
@@ -3,15 +3,15 @@
...
    <schema-version>1.0-milestone1</schema-version>
-   <short-name>metaschema-group-as</short-name>
+   <short-name>metaschema-group-as-bounded</short-name>
    <namespace>http://csrc.nist.gov/ns/metaschema/unit-test/group-as-array-bounded</namespace>
    <json-base-uri>http://csrc.nist.gov/ns/metaschema/unit-test/group-as-array-bounded</json-base-uri>
    <define-assembly name="parent">
       <formal-name>parent</formal-name>
       <description>parent assembly</description>
-      <root-name>parent</root-name>
+      <root-name>group-ass-array-bounded-parent</root-name>
```
Comment (aj-stein-gsa) [inline test-suite/schema-generation/group-as/group-as-array-bounded_metaschema.xml:14]: Minor non-blocking: perhaps the future of a fun inside joke, but I assume you want to change the typo. (I personally don't mind it.)

File: test-suite/schema-generation/group-as/group-as-array-bounded_test_array-inside_PASS.json [hunk 2]
```diff
@@ -1,5 +1,5 @@
 {
-	"parent": {
+	"group-ass-array-bounded-parent": {
```
Comment (aj-stein-gsa) [inline test-suite/schema-generation/group-as/group-as-array-bounded_test_array-inside_PASS.json:2]: Ditto on typo.

File: test-suite/schema-generation/group-as/group-as-array-bounded_test_array-singleton_FAIL.json [hunk 3]
```diff
@@ -1,5 +1,5 @@
 {
-	"parent": {
+	"group-ass-array-bounded-parent": {
```
Comment (aj-stein-gsa) [inline test-suite/schema-generation/group-as/group-as-array-bounded_test_array-singleton_FAIL.json:2]: Ditto on typo.

File: test-suite/schema-generation/group-as/group-as-array-bounded_test_array-upper_FAIL.json [hunk 4]
```diff
@@ -1,5 +1,5 @@
 {
-	"parent": {
+	"group-ass-array-bounded-parent": {
```
Comment (aj-stein-gsa) [inline test-suite/schema-generation/group-as/group-as-array-bounded_test_array-upper_FAIL.json:2]: Ditto on typo.

File: test-suite/schema-generation/group-as/group-as-array-optional_metaschema.xml [hunk 5]
```diff
@@ -3,15 +3,15 @@
...
    <schema-version>1.0-milestone1</schema-version>
-   <short-name>metaschema-group-as</short-name>
+   <short-name>metaschema-group-as-array-optional</short-name>
    <namespace>http://csrc.nist.gov/ns/metaschema/unit-test/group-as-array-optional</namespace>
    <json-base-uri>http://csrc.nist.gov/ns/metaschema/unit-test/group-as-array-optional</json-base-uri>
    <define-assembly name="parent">
       <formal-name>parent</formal-name>
       <description>parent assembly</description>
-      <root-name>parent</root-name>
+      <root-name>group-ass-array-optional-parent</root-name>
```
Comment (aj-stein-gsa) [inline test-suite/schema-generation/group-as/group-as-array-optional_metaschema.xml:14]: Ditto on typo.

File: test-suite/schema-generation/group-as/group-as-array-optional_test_empty_FAIL.json [hunk 6]
```diff
@@ -1,5 +1,5 @@
 {
-	"parent": {
+	"group-ass-array-optional-parent": {
```
Comment (aj-stein-gsa) [inline test-suite/schema-generation/group-as/group-as-array-optional_test_empty_FAIL.json:2]: Ditto on typo.

File: test-suite/schema-generation/group-as/group-as-array-optional_test_valid_PASS.json [hunk 7]
```diff
@@ -1,5 +1,5 @@
 {
-	"parent": {
+	"group-ass-array-optional-parent": {
```
Comment (aj-stein-gsa) [inline test-suite/schema-generation/group-as/group-as-array-optional_test_valid_PASS.json:2]: Ditto on typo.

File: test-suite/schema-generation/collapsible/collapsible-no-op_metaschema.xml [hunk 8]
```diff
@@ -11,7 +11,7 @@
    <define-assembly name="parent">
       <formal-name>parent</formal-name>
       <description>parent assembly</description>
-      <root-name>parent</root-name>
+      <root-name>collapsible-no-op-parent</root-name>
```
Comment (aj-stein-gsa) [inline test-suite/schema-generation/collapsible/collapsible-no-op_metaschema.xml:14]: I didn't see a change on accompanying data files, so I assume this lives up to its name?

**Review by aj-stein-gsa — APPROVED** [review]
Found some minor typos, but excellent clean up work. The minor typos are easy to fix and it should move ahead. 🚢 it!

File: test-suite/schema-generation/group-as/group-as-array-bounded_metaschema.xml (hunk 1 above)
Comment (david-waltermire) [reply test-suite/schema-generation/group-as/group-as-array-bounded_metaschema.xml:14]:
```suggestion
      <root-name>group-as-array-bounded-parent</root-name>
```

File: test-suite/schema-generation/group-as/group-as-array-bounded_test_array-inside_PASS.json (hunk 2 above)
Comment (david-waltermire) [inline test-suite/schema-generation/group-as/group-as-array-bounded_test_array-inside_PASS.json:2]:
```suggestion
	"group-as-array-bounded-parent": {
```

File: test-suite/schema-generation/group-as/group-as-array-bounded_test_array-singleton_FAIL.json (hunk 3 above)
Comment (david-waltermire) [reply test-suite/schema-generation/group-as/group-as-array-bounded_test_array-singleton_FAIL.json:2]:
```suggestion
	"group-as-array-bounded-parent": {
```

File: test-suite/schema-generation/group-as/group-as-array-bounded_test_array-upper_FAIL.json (hunk 4 above)
Comment (david-waltermire) [reply test-suite/schema-generation/group-as/group-as-array-bounded_test_array-upper_FAIL.json:2]:
```suggestion
	"group-as-array-bounded-parent": {
```

File: test-suite/schema-generation/group-as/group-as-array-optional_metaschema.xml (hunk 5 above)
Comment (david-waltermire) [reply test-suite/schema-generation/group-as/group-as-array-optional_metaschema.xml:14]:
```suggestion
      <root-name>group-as-array-optional-parent</root-name>
```

File: test-suite/schema-generation/group-as/group-as-array-optional_test_empty_FAIL.json (hunk 6 above)
Comment (david-waltermire) [reply test-suite/schema-generation/group-as/group-as-array-optional_test_empty_FAIL.json:2]:
```suggestion
	"group-as-array-optional-parent": {
```

File: test-suite/schema-generation/group-as/group-as-array-optional_test_valid_PASS.json (hunk 7 above)
Comment (david-waltermire) [reply test-suite/schema-generation/group-as/group-as-array-optional_test_valid_PASS.json:2]:
```suggestion
	"group-as-array-optional-parent": {
```

File: test-suite/schema-generation/group-as/group-as-array-bounded_test_array-inside_PASS.json (hunk 2 above)
Comment (david-waltermire) [inline test-suite/schema-generation/group-as/group-as-array-bounded_test_array-inside_PASS.json]:
```suggestion
	"group-as-array-bounded-parent": {
```

File: test-suite/schema-generation/group-as/group-as-array-bounded_metaschema.xml (hunk 1 above)
Comment (david-waltermire) [inline test-suite/schema-generation/group-as/group-as-array-bounded_metaschema.xml]:
```suggestion
      <root-name>group-as-array-bounded-parent</root-name>
```

**Pull request merged**